stories/imdb/stories/filterList.js

```diff
@@ -108,7 +133,79 @@ export default () => (
                   path={['searchRoot', 'releases']}
                   format={formatYear}
                 />
-                <TermsStats path={['searchRoot', 'genreScores']} />
+                <TermsStatsTable
+                  path={['searchRoot', 'genreScores']}
+                  tableAttrs={{ style: { margin: 'auto' } }}
+                  columns={[
```

Comment:
Is there any way we can accomplish this with child components?

Reply:
Do you mean so that we can wrap ExpandableTable within a TermsStats parent component? We would have to somehow pass the node from the path of the TermStats component as a prop to ExpandableTable. I am open to suggestions.

Reply:
Something like this?

https://github.com/smartprocure/contexture-react/blob/feature/AddTableLayoutForTermsStatsType/stories/imdb/stories/filterList.js#L144-L178

Reply:
Yeah I think that looks pretty good.